docs/accounts/how-to-change-password.md

```diff
@@ -26,7 +26,7 @@ If you are not logged in:
     * Must contain at least one lowercase letter (a–z)
     * Must contain at least one uppercase letter (A–Z)
     * Must contain at least one number (0–9)
-    * May contain special characters ( !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~)
+    * May contain special characters ( !#$%&()*+,-./:;=>?@[\]^_{|}~)
```

Comment:
https://csc-guide-preview.rahtiapp.fi/origin/penikune-password-requirements/accounts/how-to-change-password/

The backwards slash is not visible in the rendered webpage, or is it used to escape the `]` character?

Reply:
You are right, backslashes need to be escaped in Markdown. Done.